### ISSUE TITLE
Release Log

### ISSUE THREAD

**Issue by Rekhyt370** [jerryzhou196/seepreviousratings#11]
Hey!
Could you write down the changes/fixes of each new update somewhere? Something simple like "date: changed xx" would be great to keep track of the updates.
Else I am always left wondering what is new after Anki suggests that I update "see previous card ratings".

**Comment by Sowa0** [comment]
I agree with the above. At least on github, possibly ankiweb as well.

**Comment by jerryzhou196** [comment]
For sure! I will definitely put up an update log as it's probably sort of suspicious how there's like a new update every week 😅. Thanks for bringing this up @Rekhyt370 @Sowa0
